main.go

```diff
@@ -87,7 +92,8 @@ func getPackages() []*Package {
 		sha256 := resp["sha256"].(string)
 
 		if sha256 == "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5" {
-			panic("Empty sha256")
+			fmt.Println(fmt.Sprintf("Bad SHA256 for %s %s %s", goPackagePath, repoRoot.Repo, rev))
+			//panic("Empty sha256")
```

Comment:
Why would you not want to panic on a bad sha?

Reply:
In my case I let it finish then manually resolved the ones that failed - it helped me understand why they failed as well.

Reply:
I don't want to change the default as I think the current behaviour is better in general.
I'll happily accept a flag like `--keep-going` or something similar that prints warnings rather than panics.

Reply:
OK 👍 I'll try and implement it